<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SEPA QR generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
    background-color: #303841;
}
#cSymbol, #vSymbol, #sSymbol, #aSymbol{
background-color:  #303841;
}

QPushButton{
 background-color: #00ADB5;
    color: #EEEEEE;
    font-size: 26px;
    font-weight: bold;
    border-radius: 5px;
    padding: 2px;

}
QLabel{
    font-weight: bold;
    color: #EEEEEE;
    font-size: 15px;
}

QLineEdit, QDoubleSpinBox{
    color: #3A4750;
    height: 24px;
    font-size: 18px;
    border-radius: 5px;
    font-weight: bold;
    padding: 3px;
}
#settingsBtn{
	 background-color: #303841;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="saveBtn">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Save QR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copyBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>500</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy QR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate QR</string>
    </property>
   </widget>
   <widget class="QWidget" name="cSymbol" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="inputCS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>4 numbers</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Constant Symbol</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="vSymbol" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#sSymbol, vSymbol, cSymbol{
background-color: lightgrey;
}</string>
    </property>
    <widget class="QLineEdit" name="inputVS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Max. of 10 characters</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelVS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Variable Symbol</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sSymbol" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="inputSS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Max. of 10 numbers</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Specific Symbol</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelQR">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>470</width>
      <height>470</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>470</width>
      <height>470</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>470</width>
      <height>470</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="aSymbol" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>180</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QDoubleSpinBox" name="value">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>150</width>
       <height>30</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string>â‚¬</string>
     </property>
     <property name="minimum">
      <double>1.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999999999.989999771118164</double>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="settingsBtn">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>clogwheel.svg</normaloff>clogwheel.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
